README.md

```diff
@@ -1 +1,10 @@
 # soft_quota
+soft_quota.py will run a check against a Qumulo cluster running Qumulo Core 1.2.9 or later. Requires python 2.7. 
+The script will generate a csv file containing the quota name and the current usage and will email if the quota is exceeded.
+
+Credentials for the Qumulo cluster must be provided in a credentials.txt file. There is a sample file provided. Format is space delimited, 
```

Comment:
this comment is out of date (and I don't have perms to push to your master, so I can't fix it).  In future you should be able to just create a branch directly on the project in the Qumulo repo, this is a little easier to review :).